storage/src/adb/any/fixed/mod.rs

```diff
@@ -1580,177 +1637,4 @@ pub(super) mod test {
-
-            let mut db = AnyTest::init(context.clone(), db_config.clone())
-                .await
-                .unwrap();
-
-            const NUM_OPERATIONS: u64 = 500;
-            for i in 0..NUM_OPERATIONS {
-                let key = Sha256::hash(&i.to_be_bytes());
-                let value = Sha256::hash(&(i * 1000).to_be_bytes());
-                db.update(key, value).await.unwrap();
-
-                // Commit periodically to advance the inactivity floor
-                if i % 100 == 99 {
-                    db.commit().await.unwrap();
-                }
-            }
-
-            // Final commit to establish the inactivity floor
-            db.commit().await.unwrap();
-
-            // Get the root digest
-            let original_root = db.root(&mut hasher);
-
-            // Verify the pruning boundary is correct
-            let oldest_retained = db.oldest_retained_loc().unwrap();
-            let inactivity_floor = db.inactivity_floor_loc;
-            assert_eq!(
-                oldest_retained,
-                inactivity_floor.saturating_sub(db_config.pruning_delay)
-            );
-
-            // Get proof of items below inactivity floor but after pruning boundary
-            let proof_start = oldest_retained;
-            let proof_end = std::cmp::min(inactivity_floor, oldest_retained + 10);
-            let max_ops = proof_end - proof_start;
-
-            let (original_proof, original_ops) = db.proof(proof_start, max_ops).await.unwrap();
-
-            // Verify the proof works
-            assert!(verify_proof(
-                &mut hasher,
-                &original_proof,
-                proof_start,
-                &original_ops,
-                &original_root
-            ));
-
-            // Close and reopen the database
-            db.close().await.unwrap();
-            let db = AnyTest::init(context.clone(), db_config).await.unwrap();
-
-            // Confirm root is identical after restart
-            let reopened_root = db.root(&mut hasher);
-            assert_eq!(original_root, reopened_root,);
-
-            // Get proof of items below inactivity floor again
-            let (reopened_proof, reopened_ops) = db.proof(proof_start, max_ops).await.unwrap();
-
-            // Verify the proof still works and is identical
-            assert_eq!(original_proof.size, reopened_proof.size);
-            assert_eq!(original_proof.digests, reopened_proof.digests);
-            assert_eq!(original_ops, reopened_ops);
-
-            assert!(verify_proof(
-                &mut hasher,
-                &reopened_proof,
-                proof_start,
-                &reopened_ops,
-                &reopened_root
-            ));
-
-            db.destroy().await.unwrap();
-        });
-    }
-
-    // Test that databases with different `pruning_delay` values generate the same root.
-    #[test_traced("WARN")]
-    fn test_any_db_different_pruning_delays_same_root() {
```

Comment:
Another case where it may be nice to retain a variant of this test (that ensures pruning doesn't impact root).

Reply:
this is tested on L968.